addOns/oast/src/main/java/org/zaproxy/addon/oast/internal/MessageEntity.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.oast.internal;
+
+import java.sql.Timestamp;
+import javax.jdo.annotations.Cacheable;
+import javax.jdo.annotations.Column;
+import javax.jdo.annotations.IdGeneratorStrategy;
+import javax.jdo.annotations.PersistenceCapable;
+import javax.jdo.annotations.Persistent;
+import javax.jdo.annotations.PrimaryKey;
+import org.datanucleus.api.jdo.annotations.CreateTimestamp;
+import org.parosproxy.paros.network.HttpBody;
+import org.parosproxy.paros.network.HttpMessage;
+
+@Cacheable("false")
```

Comment:
I noticed this on both things. It's probably a different cache but more so I understand. Cache is off here but the SQL creates it as a cached table?

Reply:
This is for L2 cache, which is not the same as the underlying database.